.gitignore

```diff
@@ -25,3 +25,4 @@ bazel-*
 .bazelproject
 .aswb
 *.pb
+coverage_reports
```

Comment:
Is this generated by Bazel? Otherwise, where does it come from & why should it be ignored?

Reply:
It seems my previous comment, placed way above, was missed. Could you please take a look at it when you get a chance? It provides context for the changes I made. https://github.com/oppia/oppia-android/pull/5443#issuecomment-2186663348

Reply:
This conversation is addressed with confirmation to https://github.com/oppia/oppia-android/pull/5443#issuecomment-2192915627 by sticking to the approach of letting the script decide the output paths.